docs/operational_guide/multiple_m3db_clusters.md

```diff
@@ -0,0 +1,108 @@
+## Write to multiple M3DB clusters via M3Coordinator
+
+### Overview:
+
+Default M3 architecture has the M3Coordinator writing to and aggregating meterics from a single M3DB cluster. To map a single coordinator to more than one M3DB cluster, follow the below instructions. 
```

Comment:
should probably be `... aggregating and writing metrics to ...`, also probably `instructions below` vs `below instructions`